Jenkinsfile

```diff
@@ -113,8 +138,12 @@ try {
     }
 
     parallel(tasks)
-
     common.setResultIfNotSet(Constants.JSUCCESS)
+    
+    if (env.BRANCH_NAME == 'master' && releaseBuild == 'Yes') {
```

Comment:
This variable is no longer defined. Please make this a build parameter.

Reply:
Example here: 

https://github.eng.tenable.com/Product/kafka-connect-configurator/blob/32cf403cc6153cfcbe3e7e8c25c9c5b776d0a6cc/Jenkinsfile#L10